packages/perseus-core/src/data-schema.ts

```diff
@@ -346,8 +345,6 @@ export type MatrixWidget = WidgetOptions<'matrix', PerseusMatrixWidgetOptions>;
 // prettier-ignore
 export type MeasurerWidget = WidgetOptions<'measurer', PerseusMeasurerWidgetOptions>;
 // prettier-ignore
-export type MockWidget = WidgetOptions<'mock-widget', MockWidgetOptions>;
```

Comment:
I moved all types related to the MockWidget into its directory so that we don't have test pieces intermingled/shipped with our production code.